src/ukify/ukify.py

```diff
@@ -450,6 +450,21 @@ class PeError(Exception):
 def pe_add_sections(uki: UKI, output: str):
     pe = pefile.PE(uki.executable, fast_load=True)
 
+    # Old stubs do not have the symbol/string table stripped, even though image files should not have one.
+    symbol_table = pe.FILE_HEADER.PointerToSymbolTable
+    if symbol_table != 0:
```

Comment:
Omitting what is checked against for things that coerce to bools is usually considered more pythonic.
```suggestion
    symbol_table = pe.FILE_HEADER.PointerToSymbolTable
    if symbol_table:
```
and since, I think, 3.9 is the lower bound for the Python version this could be written aus
```suggestion
    if symbol_table := pe.FILE_HEADER.PointerToSymbolTable:
```